mainnet-contracts/src/interface/Connext/IConnext.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+interface IConnext {
+    // ============ BRIDGE ==============
+
+    function xcall(
```

Comment:
Please remove underscores from the parameters. We want to use named parameters in our contracts, and _varName is not our convention